api/views.py

```diff
@@ -622,7 +616,7 @@ def import_activities(request):
         res['count'] = len(res['users'])
     except (ImportException, UnicodeDecodeError) as e:
         res['error'] = ''.join(traceback.TracebackException.from_exception(e).format())
-    except:
+    except Exception:
```

Comment:
```suggestion
    except BaseException:
```
Otherwise this could in theory change the behavior :)